.github/workflows/build-and-test.yml

```diff
@@ -16,8 +16,8 @@ jobs:
           - gcc
           - clang
         tracing:
-          - LIBRAFT_TRACE=1
-          - NOLIBRAFT_TRACE=1
+          - LIBRAFT_TRACE=8
+          - NOLIBRAFT_TRACE=8
```

Comment:
Do we actually consult this env var anywhere?

Reply:
It's a bit of a hack to test tracing on/off in a matrix https://github.com/canonical/raft/blob/a60437d43d566f5dab8e8e80d48f8609dc941a30/.github/workflows/build-and-test.yml#L45 `NOLIBRAFT_TRACE` does not enable tracing :-)

Reply:
```
[14:37:09] dqlite@raft git:(trace-fix)$ gg NOLIBRAFT_TRACE
.github/workflows/build-and-test.yml:20:          - NOLIBRAFT_TRACE=8
```

Let me delete `NOLIBRAFT_TRACE` at all. Any objections?